.github/configs/lintconf.yaml

```diff
@@ -0,0 +1,49 @@
+---
+rules:
+  braces:
+    level: error
+    min-spaces-inside: 0
+    max-spaces-inside: 0
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
+  brackets:
+    level: error
+    min-spaces-inside: 0
+    max-spaces-inside: 0
+    min-spaces-inside-empty: -1
+    max-spaces-inside-empty: -1
+  colons:
+    level: error
+    max-spaces-before: 0
+    max-spaces-after: 1
+  commas:
+    level: error
+    max-spaces-before: 0
+    min-spaces-after: 1
+    max-spaces-after: 1
+  comments:
+    level: error
+    require-starting-space: true
+    min-spaces-from-content: 2
+  document-end: disable
+  document-start: disable           # No --- to start a file
+  empty-lines:
+    max: 2
+    max-start: 0
+    max-end: 0
+  hyphens:
+    level: warning
+    max-spaces-after: 1
+  indentation:
+    level: warning
```

Comment:
Why not `error`?

Reply:
@shimoncohen because i don't want to fail on everything at the very beginning

Reply:
I see, you want the developer to see all of the warnings so he could fix them all at once?

Reply:
@shimoncohen changed to `error`

Reply:
Why? I was asking to understand. If that was the reason then it sounds like the right choice.

Reply:
@shimoncohen yes, we want it to be error so the developers will fix it, if it'll be warning it'll be never fixed

Reply:
😂 That's true, I just thought that displaying all of the errors at once is better than failing over and over, but they should run this locally anyway so it checks out.

Reply:
@shimoncohen so we can mart this as resolved?

Reply:
Yup